src/pages/tap.tsx

```diff
@@ -23,6 +26,15 @@ export default function Tap() {
     [router]
   );
 
+  // Save the newly tapped location to local storage and redirect to their profile
+  const processLocationTap = useCallback(
```

Comment:
Why does this need to be a callback? Not blocking merge, just curious

Reply:
Ah I see, so this works when you redirect to LoginForm?

Reply:
yup!